dnadoh/synthesize.py

```diff
@@ -335,6 +388,7 @@ def main():
     """Main driver."""
     options = parse_args()
     random.seed(options.seed)
+    np.random.seed(options.seed)
```

Comment:
should we update to replace our existing use of `random` with `np.random` in all cases?

Reply:
Hey @brodie-brodes this was what I was thinking about: https://github.com/deepgenomics/DGPlot/pull/78#discussion_r890179839

Looks like I ended up creating a random number generator and then any time in the future where I called `np.random`, I replaced it with the `rng` 
https://github.com/deepgenomics/DGPlot/pull/78/files#diff-227d8cea269835ed562682327d4b03bb6d59adbd7515d1eca29d35b0bafbd597R86

Reply:
CC @kevqyzhu and @helen-zhu if you were interested about this topic more

Reply:
I don't think we have to update this (replacing all `random.` calls with `np.random` in this PR, but we can make an issue for it to remember to refactor/follow up on it in a subsequent PR